llama/generation.py

```diff
@@ -79,6 +79,17 @@ def build(
         """
         model_parallel_size = 1
 
+        local_rank = int(os.environ.get("LOCAL_RANK", 0))
+
+        if torch.backends.mps.is_available():
+            print ("Using MPS")
+            device = torch.device("mps")
+            device = "mps:0"
+        else:
+            print ("Using CUDA")
+            torch.cuda.set_device(local_rank)
```

Comment:
just delete it

Reply:
need to set device backend. included cpu as well